.github/ISSUE_TEMPLATE/feature_release.md

```diff
@@ -3,7 +3,7 @@ name: Feature release
 about: Begin the finishing work for features ready to be included in a release
```

Comment:
actually would you just delete this file? we're handling this checklist in asana instead.